### ISSUE TITLE
Augment Dataset

### ISSUE THREAD

**Issue by AmielCyber** [juliareinhart/waste-classification-ml#9]
Augment dataset. Use a similar function as get_realwaste_dataloaders but in transforms.Compose add augmented options to our dataset

**Comment by juliareinhart** [comment]
Hello Amiel and Carlos, 

I am working on this change now. I am adding data augmentation with comments. I will send an update when it is complete. Thanks

-Julia

**Comment by juliareinhart** [comment]
Below are two screenshots of code from our current textbook on data augmentation. Can we please cite our current textbook for code as well? Thanks so much. 

<img width="1728" height="1117" alt="Image" src="https://github.com/user-attachments/assets/7e950dbe-6748-4bed-b8eb-564f8e7dbb44" />
<img width="1728" height="1117" alt="Image" src="https://github.com/user-attachments/assets/f1e9fc6c-bad4-46a3-a84b-4a316b53a55b" />

**Comment by juliareinhart** [comment]
I have also found ligitimate sources from PyTorch documentation that mirrors the code ChatGPT gave. Can we please include these sources in our sources section? 

1. https://docs.pytorch.org/vision/stable/transforms.html
2. https://docs.pytorch.org/vision/main/generated/torchvision.transforms.RandomResizedCrop.html
3. https://docs.pytorch.org/vision/main/generated/torchvision.transforms.RandomHorizontalFlip.html
4. https://docs.pytorch.org/vision/main/generated/torchvision.transforms.ColorJitter.html
5. https://docs.pytorch.org/vision/main/generated/torchvision.transforms.RandomPerspective.html

**Comment by juliareinhart** [comment]
(Sources Continued) 
6. https://www.codegenes.net/blog/how-to-do-random-scale-pytorch/

**Comment by juliareinhart** [comment]
Currently, data augmentation is applied to training, validation, and test data sets. Per ChatGPT and our textbook, actually we only want to perform data augmentation on the training dataset

We need:

- **training dataset:** data augmentation and normalization
- **validatation/test:** resize and normalization only

I am in the progress of implementing these changes.
